NEOSPlus/Components/Wizards/ComponentBreakdown.cs

```diff
@@ -0,0 +1,121 @@
+﻿// MIT License
+
+// Copyright (c) 2023 marsmaantje
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE. 
+
+using System.Collections.Generic;
+using BaseX;
+using FrooxEngine;
+using FrooxEngine.UIX;
+
+namespace ComponentBreakdown
+{
+    [Category("Add-ons/Wizards")]
+    public class ComponentBreakdown :
+        Component
+    {
+        public readonly SyncRef<Slot> Root;
+        protected readonly SyncRef<Slot> _listRoot;
+        protected UIBuilder listGenerator;
```

Comment:
I need to point out the access modifiers on these fields: While I need to check, IIRC these could be made private. I'll check when I get home

Reply:
Can indeed be made private, did that now